src/screens/FeedScreen/ContentPoll.js

```diff
@@ -297,7 +298,7 @@ const stylesComponent = (pollLength) =>
       height: 12,
       alignSelf: 'center',
       borderRadius: 6,
-      backgroundColor: COLORS.holyTosca,
+      backgroundColor: colors.holytosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.holyTosca` to `colors.holytosca`. Make sure that `colors.holytosca` is defined and is the intended color.
```diff
- backgroundColor: COLORS.holyTosca
+ backgroundColor: colors.holytosca
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->